ambry-network/src/test/java/com.github.ambry.network/SSLBlockingChannelTest.java

```diff
@@ -116,7 +117,7 @@ public void testWrongPortConnection()
       channel.connect();
       fail("should have thrown!");
     } catch (IOException e) {
-      assertEquals(e.getMessage(), "Connection refused");
+      assertTrue(e.getMessage().contains("Connection refused"));
```

Comment:
In mac, I got an error as follows:
com.github.ambry.network.SSLBlockingChannelTest > testWrongPortConnection FAILED
    org.junit.ComparisonFailure: expected:<Connection refused[ (Connection refused)]> but was:<Connection refused[]>
        at org.junit.Assert.assertEquals(Assert.java:123)
        at org.junit.Assert.assertEquals(Assert.java:145)
        at com.github.ambry.network.SSLBlockingChannelTest.testWrongPortConnection(SSLBlockingChannelTest.java:119)

hence made this change